components/backgroundlayers/README.md

```diff
@@ -0,0 +1,7 @@
+# @spectrum-css/backgroundlayers
```

Comment:
We're sort of undecided about documentation moving forward (re: migration guidelines convo) but this might be a good place for now to document this class system once we get to a more final place with it... I also think we need documentation in Storybook and the docs site so maybe we can do some brainstorming about what that looks like. Docs site might be more straightforward than Storybook

Reply:
I did add some documentation to the readme. I'd agree that there is opportunity to keep brain storming what we want documentation to look like.